Description: Updated build classpath for Eclipse 3.4
Author: Damien Raude-Morvan <drazzib@drazzib.com>
Forwarded: no
Last-Update: 2009-08-18
--- a/org.aspectj.ajdt.core/.classpath
+++ b/org.aspectj.ajdt.core/.classpath
@@ -17,5 +17,11 @@
 	<classpathentry kind="lib" path="/lib/bcel/bcel-verifier.jar"/>
 	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER"/>
 	<classpathentry combineaccessrules="false" kind="src" path="/org.aspectj.matcher"/>
+	<classpathentry kind="lib" path="/lib/eclipse/org.eclipse.equinox.common.jar"/>
+	<classpathentry kind="lib" path="/lib/eclipse/org.eclipse.equinox.registry.jar"/>
+	<classpathentry kind="lib" path="/lib/eclipse/org.eclipse.core.resources.jar"/>
+	<classpathentry kind="lib" path="/lib/eclipse/org.eclipse.core.runtime.jar"/>
+	<classpathentry kind="lib" path="/lib/eclipse/org.eclipse.core.jobs.jar"/>
+	<classpathentry kind="lib" path="/lib/eclipse/org.eclipse.osgi.jar"/>
 	<classpathentry kind="output" path="bin"/>
 </classpath>
